docs/dev/test-configuration-options.md

```diff
@@ -707,73 +921,87 @@ You can find your `accessKey` value under **Account** > **User Settings**.
+MutableCapabilities capabilities = new MutableCapabilities();
+//...
+MutableCapabilities sauceOptions = new MutableCapabilities();
+sauceOptions.setCapability("custom-data", "{'release': '1.0','commit': '0k392a9dkjr','staging': true,'execution_number': 5,'server': ‘test.customer.com'}");
+capabilities.setCapability("sauce:options", sauceOptions);
 ```
 
 ---
+
 ### `public`
-<p><small>| STRING |</small></p>
+
+<p><small>| OPTIONAL | STRING | <span className="sauceDBlue">Desktop and Virtual Devices Only</span> |</small></p>
 
 We support several test/job result visibility levels, which control who can view the test details. The visibility level for a test can be set manually from the test results page, but also programmatically when starting a test or with our REST API. For more information about sharing test results, see the topics under [Sharing the Results of Sauce Labs Tests](/test-results/sharing-test-results).
 
 Available visibility modes are:
-* **public**:
-  * Accessible to everyone.
-  * May be listed on public web pages and indexed by search engines.
-* **public restricted**:
-  * Share your job's results page and video, but keeps the logs only for you.
-  * Hides the fancy job log.
-  * Prohibits access to the raw Selenium log so that anonymous users with the link will be able to watch the video and screen shots, but won't be able to see what's being typed and done to get there.
-* **share**:
-  * Only accessible to people with a valid link.
-  * Not listed on publicly available pages on Sauce Labs.
-  * Not indexed by search engines.
-* **team**:
-  * Best option if you want to share your jobs with other team members that were created as a sub-accounts of one parent account.
-  * Only accessible to people under the same root account as you.
-* **private**:
-  * Best option if you don't want to share your test results page and video with anyone.
-  * Only you (the owner) will be able to view assets and test results page.
+
+- **public**:
+  - Accessible to everyone.
+  - May be listed on public web pages and indexed by search engines.
+- **public restricted**:
+  - Share your job's results page and video, but keeps the logs only for you.
+  - Hides the fancy job log.
+  - Prohibits access to the raw Selenium log so that anonymous users with the link will be able to watch the video and screen shots, but won't be able to see what's being typed and done to get there.
+- **share**:
+  - Only accessible to people with a valid link.
+  - Not listed on publicly available pages on Sauce Labs.
+  - Not indexed by search engines.
+- **team**:
+  - Best option if you want to share your jobs with other team members that were created as a sub-accounts of one parent account.
+  - Only accessible to people under the same root account as you.
+- **private**:
+  - Best option if you don't want to share your test results page and video with anyone.
+  - Only you (the owner) will be able to view assets and test results page.
```

Comment:
```suggestion

* **public**:
  * Accessible to everyone.
  * May be listed on public web pages and indexed by search engines.
* **public restricted**:
  * Share your job's results page and video, but keeps the logs only for you.
  * Hides the fancy job log.
  * Prohibits access to the raw Selenium log so that anonymous users with the link will be able to watch the video and screenshots, but won't be able to see what's being typed and done to get there.
* **share**:
  * Only accessible to people with a valid link.
  * Not listed on publicly available pages on Sauce Labs.
  * Not indexed by search engines.
* **team**:
  * Best option if you want to share your jobs with other team members that were created as a sub-accounts of one parent account.
  * Only accessible to people under the same root account as you.
* **private**:
  * Best option if you don't want to share your test results page and video with anyone.
  * Only you (the owner) will be able to view assets and test results page.
```